src/v2/Apps/FairOrginizer/FairOrganizerApp.tsx

```diff
@@ -56,6 +55,7 @@ export const FairOrganizerAppFragmentContainer = createFragmentContainer(
       fragment FairOrganizerApp_fairOrganizer on FairOrganizer {
         name
         slug
+        about(format: HTML)
```

Comment:
Are other formats available? I worry a little that me might end up having tags or escaped characters in our descriptions

Reply:
They are available but we use [truncate](https://www.npmjs.com/package/trunc-html) to not escape the tags 😬 
I avoided using plain because one of the fragments uses `about` with HTML, so would cause a conflict. Anyways `PLAIN` also has special characters to be removed with truncate, I can explore changing to it if you want, np